dotnet-web/README.md

```diff
@@ -0,0 +1,36 @@
+---
+title: .NET
+description: Set up a CI pipeline for a .NET web app with restore, test and build commands
+tags: ["CI", ".NET"]
+author: Buildkite
+---
+
+# CI for .NET with Buildkite
+
+This template gives you a continuous integration (CI) pipeline that automates installing dependencies, run unit test and build.
+
+At a glance:
+
+- For .NET projects
```

Comment:
Following the others, let's link to the relevant .NET site.

Reply:
Updated README and added links https://github.com/buildkite/templates/pull/22/commits/bda5b8ca6cb30710b373092fc3d8d8d858b73673#diff-5fca869b4e5d00d2ce2ec8c059f2de58ea9e0e3cb3cd7f311bf0749d3c28098fR14